README.md

```diff
@@ -27,6 +27,18 @@ Proxy your requests just by prefixing your request URL with: `https://non-cors.h
 ```http request
 GET https://joke-api-strict-cors.appspot.com/jokes/ten    ==>    GET https://non-cors.herokuapp.com/https://joke-api-strict-cors.appspot.com/jokes/ten
 
+``` 
+```http request
+POST https://joke-api-strict-cors.appspot.com/jokes/add    ==>    GET https://non-cors.herokuapp.com/https://joke-api-strict-cors.appspot.com/jokes/add
+
+``` 
+```http request
+PUT https://joke-api-strict-cors.appspot.com/jokes/add/:id    ==>    GET https://non-cors.herokuapp.com/https://joke-api-strict-cors.appspot.com/jokes/add/:id
+
+``` 
+```http request
+DELETE https://joke-api-strict-cors.appspot.com/jokes/:id/delete    ==>    GET https://non-cors.herokuapp.com/https://joke-api-strict-cors.appspot.com/jokes/:id/delete
```

Comment:
How come a `POST, PUT or DELETE ` request be converted to a `GET` and I don't think `joke-api-strict-cors.appspot.com` API supports a `/jokes/add` path ???